features.json

```diff
@@ -82,11 +82,11 @@
 				"bulkMemory": true,
```

Comment:
a few lines above this one there is `"version": "78",` which I think should be updated to 79?

Reply:
Yeah, see my comment above, now resolved :)